benchmarks/2019.json

```diff
@@ -4283,6 +4337,24 @@
       9.52
     ]
   },
+  {
+    "measureId": "419",
```

Comment:
Past:
```
{"level":"debug","message":"measure 419 2017 submission method registry single_performance_rate_measurements: 1022 measurement-stratas","timestamp":"2019-01-04T00:42:12.424Z"}
{"level":"debug","message":"A total of 163 measurements may be in decile calculation for measure 419 registry 4","timestamp":"2019-01-04T00:42:12.424Z"}
{"level":"debug","message":"Success! measure 419 has 163 qualified TIN-NPIs reporting for submission method registry","timestamp":"2019-01-04T00:42:12.425Z"}
```

Present
```
{"level":"debug","message":"measure 419 2017 submission method registry single_performance_rate_measurements: 1022 measurement-stratas","timestamp":"2019-09-05T22:43:35.907Z"}
{"level":"debug","message":"Number of validated measurements: 163 \n          Number of duplicate measurements that are valid: 0 \n          Decile calculation for measure 419 registry 4","timestamp":"2019-09-05T22:43:35.907Z"}
{"level":"debug","message":"Success! measure 419 has 163 qualified TIN-NPIs reporting for submission method registry","timestamp":"2019-09-05T22:43:35.907Z"}
```
Need to see why this wasnt included

Reply:
Removed here:
https://github.com/CMSgov/qpp-measures-data/pull/269/files#diff-19cc2b2b0a838f96141210f2b77688c9L3913